Sources/SecureXPC/Server/XPCMachServer.swift

```diff
@@ -12,7 +12,11 @@ import Foundation
 /// In the case of this framework, the XPC Service is expected to be communicated with by an `XPCMachClient`.
 internal class XPCMachServer: XPCServer, NonBlockingStartable {
     
+    /// Name of the service.
     private let machServiceName: String
+    /// Receives new incoming connections, created once the server is started.
+    private var machService: xpc_connection_t?
```

Comment:
This isn't actually necessary for supporting a target queue. It resolves an issue with the non-blocking start I discovered while working on this PR.